src/regulamin.html

```diff
@@ -31,15 +31,49 @@
 
   <body>
     <script type="text/javascript">
-(function(window, document, dataLayerName, id) {
```

Comment:
Is this change intentional?

Reply:
build autoformats code. something updated with new opinions. This is fine